File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/AspNetUserRol.vb
Imports System.Collections.Generic

Public Class AspNetUserRol
    Public Shared Function Add(value As Models.AspNetUserRol) As Integer
        Return Data.AspNetUserRol.Add(value)
    End Function

    Public Shared Function Edit(value As Models.AspNetUserRol) As Integer
        Return Data.AspNetUserRol.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.AspNetUserRol.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.AspNetUserRol
        Return Data.AspNetUserRol.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.AspNetUserRol
        Return Data.AspNetUserRol.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.AspNetUserRol)
        Return Data.AspNetUserRol.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171704/Business/AspNetUserRol.cs
using System.Collections.Generic;

namespace Business
{
    public class AspNetUserRol
    {
	    private static Data.AspNetUserRol obj = new Data.AspNetUserRol();

        public static int Add(Models.AspNetUserRol value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.AspNetUserRol value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.AspNetUserRol GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.AspNetUserRol GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.AspNetUserRol> GetAll()
        {
            return obj.GetAll();
        }
    }
}